corehq/apps/builds/views.py

```diff
@@ -186,7 +186,7 @@ def import_build(request):
             }, status_code=400)
 
         build = CommCareBuild.create_from_zip(
-            StringIO(r.content), version, build_number)
+            io.BytesIO()(r.content), version, build_number)
```

Comment:
I don't think `BytesIO` instances are callable. The extra parens looks like a typo.

Reply:
whoops - thanks